docs/06-twiliointegration-python.md

```diff
@@ -30,15 +30,13 @@ if __name__ == '__main__':
 Create a [Dockerfile](../serving/twilio/python/Dockerfile) for the image:
 
 ```dockerfile
-FROM python:3.7
+FROM python:3.7-slim
```

Comment:
@PicardParis what is the advantage of using slim over alpine? Would it make sense to use alpine instead to keep the image as small as possible?

Reply:
Here are the main reasons I chose the Slim flavor as a trade-off:
- Size (for a HelloWorld)
  - 273 MB smaller than the full Debian
  - Only 25 MB larger than Alpine
- Simplicity & compatibility
  - Keep the Dockerfile simple for compiled packages (e.g. gRPC)
  - Avoid some issues with binaries on Alpine (musl vs libc)

| HelloWorld Image    |     Size |
| :------------------ | -------: |
| `python:3.7`        | 334.4 MB |
| `python:3.7-slim`   |  61.3 MB |
| `python:3.7-alpine` |  36.8 MB |